lib/parsers.js

```diff
@@ -71,8 +104,13 @@ const generateObjectProps = ($, topUL, api) => {
           parameters = generateObjectProps($, innerParams, api, true)
         }
       }
+      if (ret.type === 'String') {
+        possibleValues = getPossibleValues($, el, sanitizeDescription($(el).html().replace(/<code>/g, '`').replace(/<\/code>/g, '`')), api)
```

Comment:
This replacing-code-with-backticks exists somewhere else, IIRC. Can you generalize it for reuse?